src/components/views/editView/mechanisms/NodeSelection.js

```diff
@@ -1,13 +1,13 @@
 import * as React from "react";
 import { Box, Button } from "@mui/material";
-import PortWigetComp from "./PortWidgetComp";
+// import PortWigetComp from "./PortWidgetComp";
 
 class NodeSelection extends React.Component {
   render() {
-    const { classes, node, node: { options }, engine, text, changeVisibility } = this.props;
-    const selectionPoint = '-0.375rem';
-    const IN = 'in';
-    const OUT = 'out';
+    const { text, changeVisibility } = this.props;
+    // const selectionPoint = '-0.375rem';
```

Comment:
@ddelpiano I suggest to remove this since it's not a comment and it's not code (I mean cleanup)

Reply:
I agree with you but I would like first to verify with Emeka/Vidhya since they did this part, I will create a card for this so that I avoid to remove stuff that it was parked for unknown reasons to me.

https://metacell.atlassian.net/browse/PSYNEU-13